src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php

```diff
@@ -463,7 +463,9 @@ protected function isGrantedOr403(RequestConfiguration $configuration, $permissi
     protected function findOr404(RequestConfiguration $configuration)
     {
         if (null === $resource = $this->singleResourceProvider->get($configuration, $this->repository)) {
-            throw new NotFoundHttpException();
+            $message = sprintf('The "%s" has not been found', $this->metadata->getHumanizedName());
+
+            throw new NotFoundHttpException($message);
```

Comment:
Why don't we just inject the result of `sprintf` here instead of creating temporary variable just to inject it in the next line? #universityStyle 🌮